src/hacbs/components/Commits/tabs/CommitDetails/CommitComponentFactory.tsx

```diff
@@ -0,0 +1,30 @@
+import {
+  ComponentFactory,
+  GraphComponent,
+  ModelKind,
+  SpacerNode,
+  TaskEdge,
+  withPanZoom,
+} from '@patternfly/react-topology';
+import { NodeType } from '../../../ApplicationDetails/tabs/overview/visualization/const';
+import CommitWorkflowNode from '../../../ApplicationDetails/tabs/overview/visualization/nodes/CommitWorkflowNode';
+
+export const commitComponentFactory: ComponentFactory = (kind: ModelKind, type: string) => {
+  switch (kind) {
+    case ModelKind.graph:
+      return withPanZoom()(GraphComponent);
```

Comment:
pan zoom is fine for now until UX gives a final decision if this needs to be removed in lieu of scroll bars.

Reply:
Inconsistent with the application overview graph. Whichever way is finally decided, these should be made consistent.